packages/types/src/Delegation.ts

```diff
@@ -9,18 +9,19 @@ import type { ICType } from './CType'
 import type { IDidDetails } from './DidDetails'
 
 /* eslint-disable no-bitwise */
-export enum Permission {
-  ATTEST = 1 << 0, // 0001
-  DELEGATE = 1 << 1, // 0010
-}
+export const Permission = {
+  ATTEST: 1 << 0, // 0001
+  DELEGATE: 1 << 1, // 0010
+} as const
+export type Permissions = typeof Permission[keyof typeof Permission]
```

Comment:
wow. not easy to come up with that :D

Reply:
seems to just resolve to a simple `number`. But maybe future TS versions would support more explicit typing here.

Reply:
Interesting, it works better for strings and other types. Maybe this is already a bug in TS.